src/ark/segmentation/fiber_segmentation.py

```diff
@@ -16,6 +16,7 @@
 from skimage.measure import regionprops_table
 from skimage.morphology import remove_small_objects
 from skimage.segmentation import watershed
+from tqdm.notebook import tqdm
```

Comment:
we should use `from tqdm.auto import tqdm` so `tqdm` can decide where it's running.